docs/modules/operation/pages/device-config-backup/configuration.adoc

```diff
@@ -31,19 +40,18 @@ config:update
 Devices may also push configuration through TFTP manually whenever their configuration changes.
 This is not enabled by default; you must enable the sink for device configuration.
```

Comment:
I would suggest cleaning up this section to use wording similar to what was just added to the ticketing section. 
```
. To make the device config sink start automatically on future service starts, add the line `opennms-deviceconfig-sink` to the file `$\{OPENNMS/MINION_HOME}/etc/featuresBoot.d/device-config.boot`.
(Create the file if it does not exist.)
. To make the device config sink start now, log in to the Karaf shell and run the following command to load the feature:

 feature:install opennms-deviceconfig-sink
 ```